pkg/packagekit/updates.jsx

```diff
@@ -504,10 +504,12 @@ class RestartServices extends React.Component {
     render() {
         let body;
         if (this.props.tracerRunning) {
-            body = (<>
-                <div className="spinner spinner-xs spinner-inline" />
-                <p>{_("Reloading the state of remaining services")}</p>
-            </>);
+            body = (
+                <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
+                    <Spinner isSvg size="sm" />
+                    <p>{_("Reloading the state of remaining services")}</p>
+                </Flex>
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18688-20230421-132859-4e9b603d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)